### PR TITLE
Erivas28 edits

### PR DESCRIPTION
**Pull request by erivas28**
Navbar feature

### CODE REVIEW

File: src/main/webapp/styles.css.map [hunk 1]
```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["styles.scss","styles.css"],"names":[],"mappings":"AAAA;EACI,iBAAA;EACA,kBAAA;EACA,YAAA;ACCJ;;ADEE;EACE,gBAAA;ACCJ","file":"styles.css"}
```
Comment (Renaud2002) [inline src/main/webapp/styles.css.map:1]: Hi @erivas28 if you don't mind me asking what do the styles.css.map file do?

**Review by Renaud2002 — COMMENTED** [review]
Everything else looks good, great job.

**Review by Renaud2002 — APPROVED** [review]
👍

File: src/main/webapp/styles.css.map (hunk 1 above)
Comment (SantiM1) [reply src/main/webapp/styles.css.map:1]: Yes, I have the same question. Is it an automatically created file?

**Review by SantiM1 — APPROVED** [review]
It's looking good

File: src/main/webapp/styles.css.map (hunk 1 above)
Comment (erivas28) [reply src/main/webapp/styles.css.map:1]: I believe it is automatically created file, because I never manually created that :)

**Pull request merged**